docs/api/native-image.md

```diff
@@ -51,6 +51,9 @@ Check the _Size requirements_ section in [this article][icons].
 
 [icons]: https://learn.microsoft.com/en-us/windows/win32/uxguide/vis-icons
 
+EXIF meta-data is currently not supported and will not be taken into account during
+image encoding and decoding.
```

Comment:
```suggestion
:::note

EXIF metadata is currently not supported and will not be taken into account during
image encoding and decoding.

:::
```

Making the call out a little more prominent, and changing spelling to "metadata" without the hyphen so it's more searchable.